irods/test/scripts/test007_pam_features_in_new_auth_framework.bats

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bats
+#
+# Test creation of .irodsA for iRODS pam_password authentication, this time purely internal to the PRC
+# library code.
+
+. "$BATS_TEST_DIRNAME"/test_support_functions
+PYTHON=python3
+
+# Setup/prerequisites are same as for login_auth_test.
+# Run as ubuntu user with sudo; python_irodsclient must be installed (in either ~/.local or a virtualenv)
+#
+
+ALICES_ORIGINAL_PAM_PASSWORD=test123
+ALICES_NEW_PASSWORD=test_1234
```

Comment:
The two passwords are very similar. Can we invert one or use a different set of characters entirely so it's easier for the reader to spot the difference?

Reply:
Maybe. The price for being similar is that they are precise, in that they describe two similar things.  (Alice's password at two different points in the action.)

Reply:
Ok. I take it the test updates alice's password to the new one, correct?

Reply:
Yes, it's done in the chpasswd, [here](https://github.com/d-w-moore/python-irodsclient/blob/fd8ff95f6435a594318ac01a70afbe1d4a22e53c/irods/test/scripts/test007_pam_features_in_new_auth_framework.bats#L88).